ui/src/pages/frontsite/Documents/DocumentsDetails/DocumentItems/DocumentItems.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import isEmpty from 'lodash/isEmpty';
+import _has from 'lodash/has';
```

Comment:
nit: I remember that we agreed not to use `_` for the methods imports but correct me if I'm wrong

Reply:
@kprzerwa perfect because I was always confused but lately I sticked to the `_` due to the name `lodash` of it.

Reply:
> nit: I remember that we agreed not to use `_` for the methods imports but correct me if I'm wrong

mmm... I always pushed to keep the `_` so that we know that it is `lodash`. Did we really agreed that?

Reply:
Well I don't remember but whatever we agree on, update the relevant task in the issue https://github.com/inveniosoftware/invenio-app-ils/issues/757